release 1.30.0 (#1166)
Kong/deck: CHANGELOG.md

- [v0.2.0](#v020)
- [v0.1.0](#v010)

## [v1.30.0]

> Release date: 2023/01/11

### Fixes

- Correct bug when consumer-group-consumer doesn't have an username.
  [#1113](https://github.com/Kong/deck/pull/1113)

> - Improve deprecation warnings to reduce upgrade friction.

Reviewer: ```suggestion
- Improve deprecation warnings to reduce upgrade friction and show warning when reading STDIN from terminal.
```

  [#1115](https://github.com/Kong/deck/pull/1115)

> - O2K: Server ports will now be properly parsed, 32767 to 65535 are now accepted.

Reviewer: ```suggestion
- `file openapi2kong`: Server ports will now be properly parsed, 32767 to 65535 are now accepted.
```

  [apiops #105](https://github.com/Kong/go-apiops/pull/105)

### Added

> - O2K: will now generate OpenIDConnect plugins.

Reviewer: ```suggestion
- 'file openapi2kong': will now generate OpenIDConnect plugins.
```
